coala_quickstart/generation/FileGlobs.py

```diff
@@ -0,0 +1,21 @@
+from coala_utils.Question import ask_question
+
+
+def ask_glob_list(printer, prompt, default="**"):
+    """
+    Ask the user for a list of glob expressions to collect files from.
+
+    :param printer:
+        A ``ConsolePrinter`` object.
+    :param prompt:
+        Question to ask the user.
+    :param default:
+        Default value; ``**`` will match every file.
+    :return:
+        The list of glob expressions.
+    """
+    return ask_question(
+        prompt,
+        default=default,
+        printer=printer,
+        typecast=list)
```

Comment:
actually you don't need the ask_glob_list function anymore, just run ask+question directly, it's way clearer